### PR TITLE
adding another line

### PR DESCRIPTION
**Pull request by nikhileragam**
Please check and let me know

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # class1
 class1 created
 I need to add lines
+I added another line
```
Comment (nikhilreddy244) [inline README.md:4]: can you add one more line

Comment (nikhileragam) [reply README.md:4]: sure i will

Comment (nikhilreddy244) [reply README.md:4]: sure i will

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
 class1 created
 I need to add lines
 I added another line
+Adding 5th line
```
Comment (nikhilreddy244) [inline README.md:5]: Thanks

**Pull request merged**